### PR TITLE
CheckStyle in Voters cookbook

### PR DESCRIPTION
**Pull request by JakeFr**
Hello!

Apply fixes done by php-cs-fixer (Symfony level) after copy/paste this code sample in a project

### CODE REVIEW

File: cookbook/security/voters.rst [hunk 1]
```diff
@@ -147,10 +147,10 @@ would look like this::
             }
 
             // you know $subject is a Post object, thanks to supports
-            /** @var Post $post */
+            /* @var Post $post */
```
Comment (WouterJ) [inline cookbook/security/voters.rst]: this change has to be reverted. Seems like a bug in PHP CS Fixer

Comment (JakeFr) [inline cookbook/security/voters.rst]: Change made by phpdoc_to_comment fixer (Symfony level)
Docblocks should only be used on structural elements.

Comment (WouterJ) [inline cookbook/security/voters.rst]: Ah, that rule isn't used by Symfony. Please disable that fixer :)

**Comment by WouterJ** [comment]
The other change is correct, let's merge 👍

**Comment by JakeFr** [comment]
File changed

**Comment by WouterJ** [comment]
Status: Reviewed

**Comment by javiereguiluz** [comment]
👍

**Comment by WouterJ** [comment]
Thanks @JakeFr! I've merged your PR in the 2.7 version of the docs (the oldest version we still maintain) and I'll merge it up to the 3.2 docs from there.

**Pull request closed without merging**